users/forms.py

```diff
@@ -51,27 +47,21 @@ class UserRegisterForm(UserCreationForm):
     email = forms.EmailField()
 
     class Meta:
-        model = User
+        model = CustomUser
         fields = ['username', 'email', 'first_name',
                   'last_name', 'password1', 'password2']
 
 
-class ProfileUpdateForm(forms.ModelForm):
+class UserUpdateForm(forms.ModelForm):
     experience = forms.IntegerField()
     date_of_birth = forms.DateField(widget=DateInput())
     education = forms.ChoiceField(choices=EDUCATION_CHOICES_EMPTY_LABEL)
     gender = forms.ChoiceField(choices=GENDER_CHOICES_EMPTY_LABEL)
 
     class Meta:
-        model = Profile
-        fields = ['image', 'experience', 'date_of_birth', 'education',
-                  'gender']
+        model = CustomUser
```

Comment:
Generally speaking it would be better to use the [get_user_model](https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#django.contrib.auth.get_user_model) helper here instead of direct import